### PR TITLE
feat: initial template

### PR DESCRIPTION
**Pull request by josephwwilliams**

### CODE REVIEW

File: src/assets/styles/globals.css
Comment (WilderDev) [inline src/assets/styles/globals.css:1]: Not sure if we want this on the template... as not all projects will have dark mode.And it starts to get unreadable unless there is an easy way to create these values?

File: src/components/branding/LogoSquare.tsx
Comment (WilderDev) [inline src/components/branding/LogoSquare.tsx:1]: It's funny you did this. I've always created my own cn function (where you got it), but I saw you used clsx so I said what the hell.

File: src/lib/common/baseUrl.ts
Comment (WilderDev) [inline src/lib/common/baseUrl.ts:1]: This one too is a function I almost always have... but it never works on staging from vercel. We'll see if it's updated

File: README.md
Comment (WilderDev) [inline README.md:1]: Can you add how we use LazyImage component

File: tailwind.config.ts
Comment (WilderDev) [inline tailwind.config.ts:1]: Why did you get rid of my plugins here?

Comment (WilderDev) [reply tailwind.config.ts:1]: I need my :hocus class... use it more than anything else.

**Review by WilderDev — CHANGES_REQUESTED** [review]
Everythings good except for tailwind config... want to hear your reasoning

File: README.md
Comment (josephwwilliams) [reply README.md:1]: fixed in recent commit

File: src/assets/styles/globals.css
Comment (josephwwilliams) [reply src/assets/styles/globals.css:1]: removed in recent commit on shadcn ui there is a themes section that you just copy and paste those in

File: src/components/branding/LogoSquare.tsx
Comment (josephwwilliams) [reply src/components/branding/LogoSquare.tsx:1]: Yes sir shadcn ui actually creates it as a utility function whenever you initialize it

File: src/lib/common/baseUrl.ts
Comment (josephwwilliams) [reply src/lib/common/baseUrl.ts:1]: I hope it works cause the lazy image depends on it

File: tailwind.config.ts
Comment (josephwwilliams) [reply tailwind.config.ts:1]: I didn't mean too it is back now

**Review by WilderDev — APPROVED** [review]

**Pull request merged**